packages/calcite-components/src/components/pagination/pagination.stories.ts

```diff
@@ -27,6 +35,57 @@ export const simple = (): string => html`
   </calcite-pagination>
 `;
 
+const breakpoints = [475, 476, 768, 1152];
+
+const getResponsiveTemplate = ({
+  width,
+  scale,
+  totalItems,
+  pageSize,
+}: {
+  width: number;
+  scale: Scale;
+  totalItems: number;
+  pageSize: number;
+}) => {
+  return html`<strong>Width: ${width}px</strong>
+    <div style="width: ${width}px; margin: 1em 0;">
+      <calcite-pagination
+        total-items="${totalItems}"
+        page-size="${pageSize}"
+        start-item="1"
+        scale="${scale}"
+      ></calcite-pagination>
+      <calcite-pagination
+        total-items="${totalItems}"
+        page-size="${pageSize}"
+        start-item="${totalItems / 2 - Math.max(pageSize / 2, 1) + 1}"
+        scale="${scale}"
+      ></calcite-pagination>
+      <calcite-pagination
+        total-items="${totalItems}"
+        page-size="${pageSize}"
+        start-item="${totalItems - pageSize + 1}"
+        scale="${scale}"
+      ></calcite-pagination>
+    </div>`;
+};
+
+export const responsiveSmall = (): string =>
```

Comment:
The new [`createBreakpointStories`](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/.storybook/utils.tsx#L153) should help simplify this. 🎉 

This can be done in a follow-up PR.

Reply:
done!